test/routes/routes.test.js

```diff
@@ -31,10 +39,16 @@ const isOnIgnoreList = (endpoint, method) => {
 };
 
 const acceptedResults = (status, endpoint, method) => {
+	if (isOnWhitelist(endpoint)) {
+		let whitelistResult = checkWhitelist(endpoint, method, status);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['whitelistResult' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5673434)